lua/methods/coalitions.lua

```diff
@@ -21,13 +21,158 @@ local skill = {
   Player = 5
 }
 
+-- types for aircraft taking off
+local takeoffTypes = {
```

Comment:
Ground units also use the "Turning Point" type. I think this would better be named as "waypointType" (also, singular)